app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
As part of the project requirement, Let's add a method that updates the comments counter for a post.